### PR TITLE
Sign up

### PR DESCRIPTION
**Pull request by Manishsharma203**
For preview refer link:
`https://t7m71m.csb.app/pages/sign-up.html`

### CODE REVIEW

**Review by shubhmehra — APPROVED** [review]
added some suggestions

File: index.html [hunk 1]
```diff
@@ -1,11 +1,61 @@
...
+                  <div class="submenu">
+                    <input id="submenu-input" type="checkbox" />
+                    <label for="submenu-input" class="submenu-text">
+                      Submenu
+                    </label>
+                    <ul>
+                      <li>Option 1</li>
+                      <li>Option 12</li>
+                      <li>Option 123</li>
+                    </ul>
```
Comment (vikrantkode) [inline index.html:35]: Try To Remove the commented code while raising PR

File: pages/cart.html [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+                    <input id="submenu-input" type="checkbox" />
+                    <label for="submenu-input" class="submenu-text">
+                      Submenu
+                    </label>
+                    <ul>
+                      <li>Option 1</li>
+                      <li>Option 12</li>
+                      <li>Option 123</li>
+                    </ul>
+                  </div> -->
```
Comment (vikrantkode) [inline pages/cart.html:38]: same here

File: style.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+.filter-sidebar,.content-listing{
+  height: 80vh;
+  overflow-y: auto;
+}
+.whishlist-card{
+  height: 200px;
+  width: 98%;
+}
```
Comment (vikrantkode) [inline style.css:42]: Use consistent spacing.

**Review by vikrantkode — CHANGES_REQUESTED** [review]
-Some changes suggested
-Dont rise all the files at once. Try making branches and then raise
- Cant find deploy preview , if not deployed...try adding screenshots.

**Pull request merged**